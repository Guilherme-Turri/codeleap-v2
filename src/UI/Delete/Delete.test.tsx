import { screen } from '@testing-library/react'
import { Delete } from '.'
import { componentRender } from '../../render/componentRender'

describe('</Delete>', () => {
  it('should render on screen', () => {
    const { container } = componentRender(<Delete />)
    expect(screen.getByTestId('Delete')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  })
})