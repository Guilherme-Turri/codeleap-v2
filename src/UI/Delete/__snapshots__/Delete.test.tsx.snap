// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`</Delete> should render on screen 1`] = `
.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c0 {
  color: #00CBA9;
  width: 100%;
  height: 100%;
}

<div>
  <div
    class="c0"
    data-testid="Delete"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zm2.46-7.12 1.41-1.41L12 12.59l2.12-2.12 1.41 1.41L13.41 14l2.12 2.12-1.41 1.41L12 15.41l-2.12 2.12-1.41-1.41L10.59 14l-2.13-2.12zM15.5 4l-1-1h-5l-1 1H5v2h14V4z"
      />
    </svg>
  </div>
</div>
`;
